tests/MMS/elm-pb/makegrids.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
Isn't this one fixed in #843?

Reply:
I tried to remove differences between #641 and next - while #843 came from #742 
I will close this PR for now ...